unittests/qir/NVQIRTester.cpp

```diff
@@ -123,6 +124,7 @@ CUDAQ_TEST(NVQIRTester, checkQuantumIntrinsics) {
   auto q = src;
   auto ctls = __quantum__rt__array_slice_1d(qubits, 2, 1, 2);
   __quantum__qis__swap(src, tgt);
+  __quantum__qis__swap__ctl(ctls, src, tgt);
```

Comment:
Added a set of tests here before I realized we don't have a connection between controlled SWAPs in Quake down to QIR. Filed an issue here #206